ngraph/core/include/ngraph/op/gather_elements.hpp

```diff
@@ -0,0 +1,55 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v6
+        {
+            /// \brief GatherElements operation
+            ///
+            class NGRAPH_API GatherElements : public Op
```

Comment:
Hmm looks like I already saw GatherElements in another PR https://github.com/openvinotoolkit/openvino/pull/3624...

Reply:
Agree with @ilyachur. It is not a good practice because occasionally without appropriate review of this part the other PR can be merged. Otherwise, we have to replicate our efforts to review it twice.
It is okay to have red pre-commit check if some PR is dependent on the other one.

Reply:
Thanks for your reply. I will remove operation declaration from #3624.

Reply:
Removed op declaration from #3624